frontend/src/components/Sidebar.tsx

```diff
@@ -599,3 +608,84 @@ export function useSidebarItem(itemName: string | null) {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
 }
+
+function searchNameInSubList(sublist : SidebarEntry['subList'], name: string): boolean {
+  if (!sublist) {
+    return false;
+  }
+  for (let i = 0; i < sublist.length; i++) {
+    if (sublist[i].name === name){
+      return true;
+    }
+  }
+  return false;
+}
+
+function findParentOfSubList(list: SidebarEntry[], name: string | null): SidebarEntry | null{
+  if (!name) {
+    return null;
+  }
+
+  let parent = null;
+  for (let i = 0; i < list.length; i++) {
+    if (searchNameInSubList(list[i].subList, name)) {
+      parent = list[i];
+    }
+  }
+  return parent;
+}
+
+export function NavigationTabs() {
+  const history = useHistory();
+  const sidebar = useTypedSelector(state => state.ui.sidebar);
+  if (sidebar.isSidebarOpen) {
+    return null;
+  }
+
+  let defaultIndex = null;
+  const listItems = prepareRoutes();
+  let navigationItem = listItems.find((item) => item.name === sidebar.selected);
+  if (!navigationItem) {
+    const parent = findParentOfSubList(listItems, sidebar.selected);
+    if (!parent) {
+      return null;
+    }
+    navigationItem = parent;
+  }
+
+  const subList = navigationItem.subList;
+  if (!subList) {
+    return null;
+  }
+
+  function tabChangeHandler(index: number) {
+    if (!subList) {
+      return;
+    }
+    let pathname;
+
+    const url = subList[index].url;
+    if (url) {
+      pathname = generatePath(getClusterPrefixedPath(url), {cluster: getCluster()!});
+    } else {
+      pathname = createRouteURL(subList[index].name);
+    }
+    history.push({pathname});
+  }
+
+  const tabRoutes = subList.map((item: any) => {
+    return {'label': item.label, component: <></>};
+  });
+
+  defaultIndex = subList.findIndex((item) => item.name === sidebar.selected);
+  return (
+    <>
+      <Tabs
+        tabs={tabRoutes}
+        resetState={sidebar.isIconClicked}
```

Comment:
Are you sure this is needed? I don't love having this in the `Tabs` component and would prefer if we rely on an `index` property from outside.